### ISSUE TITLE
TypeError: download() got an unexpected keyword argument 'keep_cover'

### ISSUE THREAD

**Issue by TheProgramableTurtle** [nathom/streamrip#13]
```
C:\>rip -u <DEEZER_URL>
Traceback (most recent call last):
  File "c:\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python39\Scripts\rip.exe\__main__.py", line 7, in <module>
  File "c:\python39\lib\site-packages\streamrip\cli.py", line 205, in main
    cli(obj={})
  File "c:\python39\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "c:\python39\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "c:\python39\lib\site-packages\click\core.py", line 1236, in invoke
    return Command.invoke(self, ctx)
  File "c:\python39\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\python39\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "c:\python39\lib\site-packages\click\decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "c:\python39\lib\site-packages\streamrip\cli.py", line 58, in cli
    core.download()
  File "c:\python39\lib\site-packages\streamrip\core.py", line 146, in download
    item.download(**arguments, tag=True)
TypeError: download() got an unexpected keyword argument 'keep_cover'
```
Occurs when downloading a single track from Deezer, but not an entire album.

**Comment by TheProgramableTurtle** [comment]
For anyone wanting a quick fix, locate the `streamrip` library directory on your system, open `core.py` and comment out lines 127 and 128 (type a `#` at the start of each line). This stops `keep_cover` and `large_cover` from being passed to `download()`. The cover art will be kept and will be large.

**Comment by nathom** [comment]
Fixed in `v0.2.12`
